ui/lib/kv/addon/components/page/list.hbs

```diff
@@ -121,6 +123,16 @@
         </div>
       </LinkedBlock>
     {{/each}}
+    {{! Pagination }}
+    <Hds::Pagination::Numbered
+      @currentPage={{@secrets.meta.currentPage}}
+      @currentPageSize={{@secrets.meta.pageSize}}
+      @route={{this.router.currentRoute.localName}}
+      @showSizeSelector={{false}}
+      @totalItems={{@secrets.meta.total}}
+      @queryFunction={{this.paginationQueryParams}}
+      data-test-pagination
+    />
   {{else}}
```

Comment:
Pagination now always shows. Which I'm a fan of. 
![image](https://github.com/hashicorp/vault/assets/6618863/b44d764a-a557-4c5b-b5c7-2e01bec0efb4)